### DISCUSSION TITLE
도메인 등록 이후에 복사를 할 수 있는 기능을 추가하면 어떨까요?

### DISCUSSION THREAD

**Discussion by hmu332233** [ChoJongHoon/develofolio#51] [Ideas]
굉장히 사소하지만
주소를 복사해서 공유를 하는 경우가 많은 서비스의 특성상
도메인 등록 후에 복사를 할 수 있게 해주는 기능 하나가 좀 더 나은 UX를 제공해줄 수 있지 않을지 생각해봅니다..!

**Comment by ChoJongHoon** [comment]
좋은 아이디어 감사합니다! UI 구성 후 구현하겠습니다 ㅎㅎ

**Comment by ChoJongHoon** [comment]
commit: c006854148567791882993c02b9ebe90a6741431

복사 기능 추가했습니다!
